sql/spanner/inspect.go

```diff
@@ -0,0 +1,738 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package spanner
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"ariga.io/atlas/sql/internal/sqlx"
+	"ariga.io/atlas/sql/schema"
+)
+
+// A diff provides an SQLite implementation for schema.Inspector.
+type inspect struct{ conn }
+
+var _ schema.Inspector = (*inspect)(nil)
+
+// defaultSchemaNameAlias is what we map Spanner's empty schema to to enable it to be referenced in HCL representations.
+const defaultSchemaNameAlias = "default"
+
+// InspectRealm returns schema descriptions of all resources in the given realm.
+func (i *inspect) InspectRealm(ctx context.Context, opts *schema.InspectRealmOption) (*schema.Realm, error) {
+	schemas, err := i.schemas(ctx, opts)
+	if err != nil {
+		return nil, fmt.Errorf("issue in schemas(): %w", err)
```

Comment:
Error messages are almost ~identical in all drivers for consistency reasons. Let's keep them this way. 

Also, this wrapping is not needed (like in other drivers), because it's most likely `fmt.Errorf("spanner: querying schemas: %w", err)`.